packages/usa-layout-docs/src/styles/_usa-layout-docs.scss

```diff
@@ -2,16 +2,9 @@
 
 // Flexbox positioning to move sidenav below main content on small screens
```

Comment:
We should update / remove this comment now that we're no longer moving the sidenav below main content

Reply:
Updated in 219e6b3dc2dee006fccf198396d2da8f35d98214.

In the future, an update or a PR suggestion would be appreciated.